java/grpc-client/src/main/java/io/vitess/client/grpc/GrpcClientFactory.java

```diff
@@ -88,9 +90,11 @@ public GrpcClientFactory setNameResolverFactory(NameResolver.Factory value) {
    */
   @Override
   public RpcClient create(Context ctx, String target) {
+    ClientTracingInterceptor tracingInterceptor = new ClientTracingInterceptor();
```

Comment:
Can we add an enable flag for this and default to false. `public GrpcClientFactory enableOpenTracing()`
This way we can toggle it with a jdbc config as well.

Reply:
I'm not sure what you mean here. Would you mind coding up an example of what you are after?